### PR TITLE
nnbd preparation

### PR DESCRIPTION
**Pull request by vasilich6107**

### CODE REVIEW

File: test/query_generator/interfaces/interface_test.dart [hunk 1]
```diff
@@ -22,7 +22,9 @@ const query = r'''
     nodeById(id: $id) {
       id
       __typename
-      ... on User {
+      ... on User {// @dart = 2.8
```
Comment (comigor) [inline test/query_generator/interfaces/interface_test.dart:25]: is it right?

Comment (vasilich6107) [reply test/query_generator/interfaces/interface_test.dart]: nope)
fixed

**Comment by vasilich6107** [comment]
> Do you know if this will cause retrocompatibility issues on older Dart versions?

This will definitely cause issues if someone's project is too old)
I tried to use the `flutter 1.12.0` but the first error was

```
The current Dart SDK version is 2.7.0.

Because artemis depends on gql_http_link >=0.2.9 which requires SDK version >=2.7.2 <3.0.0, version solving failed.
```
The nearest flutter with 2.8+ version is `1.17.0`

**Comment by comigor** [comment]
So 1.17 (beginning of May) was the stable release just after 1.12. I think we're good with a three-month old version.

**Pull request merged**